README.md

```diff
@@ -241,6 +241,20 @@ View with will_paginate
 <%= will_paginate @products %>
 ```
 
+### Scroll API
+
+For large datasets, a scroll context is more efficient than deep pagination. Create a scroll context by passing in the desired keep-alive time and the number of records per page
+
+```ruby
+products = Product.search "milk", per_page: 10, scroll: '1m'
+```
+
+Request the next set of records in the scroll context
+
+```ruby
+next_records = products.scroll
```

Comment:
The docs state:

> The initial search request and each subsequent scroll request each return a _scroll_id, which may change with each request — only the most recent _scroll_id should be used.

So we want the scroll method to be called on the new result set each time instead of the original. I'm not sure how we want to get this across in the docs.